tests/specs/integration/BaseEntity/ChildClassSpec.cfc

```diff
@@ -0,0 +1,297 @@
+component extends="tests.resources.ModuleIntegrationSpec" appMapping="/app" {
+
+	function run() {
+		describe( "Child Class Spec", function() {
+			it( "merges parent table and child table attributes int to entity", function() {
+                var jingle = getInstance( "Jingle" ).first();
+
+                expect( jingle.isLoaded() ).toBeTrue();
+
+                var memento = jingle.getMemento();
+
+                expect( memento )
+                    .toHaveKey( "id" )
+                    .toHaveKey( "title" )
+                    .toHaveKey( "catchiness" )
+                    .toHaveKey( "createdDate" )
+                    .toHaveKey( "modifiedDate" );
+
+                expect( jingle.getId() ).notToBeNull();
+                expect( jingle.isLoaded() ).toBeTrue();
+
+            } );
+            
+            it( "can create a child entity and populate the parent and child tables", function(){
+                var newJingle = getInstance( "Jingle" ).fill(
+                    {
+                        "title" : "Give me a Break",
+                        "downloadURL" : "https://www.youtube.com/watch?v=0nkcVz1mad0",
+                        "createdDate" : now(),
+                        "modifiedDate" : now(),
+                        "catchiness" : 2
+                    }
+                ).save();
+
+                var memento = newJingle.getMemento();
+
+                expect( memento )
+                    .toHaveKey( "id" )
+                    .toHaveKey( "title" )
+                    .toHaveKey( "catchiness" )
+                    .toHaveKey( "createdDate" )
+                    .toHaveKey( "modifiedDate" );
+
+                expect( newJingle.getId() ).notToBeNull();
+                expect( newJingle.isLoaded() ).toBeTrue();
+                
+            } );
+
+            it( "can update a child entity", function(){
+
+                var newJingle = getInstance( "Jingle" ).fill(
+                    {
+                        "title" : "Give me a Break",
+                        "downloadURL" : "https://www.youtube.com/watch?v=0nkcVz1mad0",
+                        "createdDate" : now(),
+                        "modifiedDate" : now(),
+                        "catchiness" : 2
+                    }
+                ).save();
+
+                transactionSetSavePoint( 'jingle-saved' );
```

Comment:
It's probably not necessary with quick since it executes against the database immediately and doesn't pool statements, actually. With CF ORM, you need them to make sure the SQL is executed within the transaction so that the database is up to date.  I can remove them from these tests.